rfc3164/syslog_message.go

```diff
@@ -28,6 +32,12 @@ func (sm *syslogMessage) export() *SyslogMessage {
 	out := &SyslogMessage{}
 	out.ComputeFromPriority(sm.priority)
 
+	if sm.msgcountSet {
```

Comment:
No `sm.ciscoHostnameSet`...

I guess, you don't want to catch the "Cisco Hostname" part... Right?

Reply:
It just uses the regular hostname field.